src/App.js

```diff
@@ -2,13 +2,15 @@ import { Header } from "./components/Header";
 import { Routes, Route } from "react-router-dom";
 import { ReviewList } from "./components/ReviewList";
 import { SingleReview } from "./components/SingleReview";
+import{LoginPage} from './components/LoginPage'
 import { useState } from "react";
 import "bootstrap/dist/css/bootstrap.min.css";
 import "./App.css";
 
 
 function App() {
-
+  const [user, setUser] = useState(null)
```

Comment:
Not necessary, but a great opportunity to use context here as many subsequent components will behave differently based on signed in user.
Would also be nice to indicate who we are logged in as - perhaps log in button changes to "logged in as such and such"